rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -653,81 +665,115 @@ static std::string get_node_user_data(const char * node_name, const char * node_
 
 #if RMW_SUPPORT_SECURITY
 /*  Returns the full URI of a security file properly formatted for DDS  */
-char * get_security_file_URI(
-  const char * security_filename, const char * node_secure_root,
+bool get_security_file_URI(
+  char ** security_file, const char * security_filename, const char * node_secure_root,
   const rcutils_allocator_t allocator)
 {
-  char * ret;
+  char * uri = nullptr;
```

Comment:
Why not just use `security_file` directly?

Reply:
Updated in bf8ccfa